packages/components/src/components/Label/Label.tsx

```diff
@@ -31,7 +25,7 @@ const Label = React.forwardRef<HTMLLabelElement, LabelProps>(
         fontSize="fontSize10"
         fontWeight="fontWeightMedium"
         lineHeight="lineHeight10"
-        marginBottom={marginBottom}
+        marginBottom="space20"
         ref={ref}
         {...props}
```

Comment:
So, if we pass another marginBottom to `Label` component while using it, it will override this "space20", right?

Reply:
yes!